src/components/MewCheckbox/MewCheckbox.vue

```diff
@@ -5,7 +5,6 @@
       v-model="isChecked"
     />
     <span
-      @click="onToggle()"
```

Comment:
i think we want the label to be able to toggle too! unless the designs say otherwise.... maybe i can add a prop to this